contracts/refs/CoreRef.sol

```diff
@@ -9,6 +9,8 @@ import "@openzeppelin/contracts/security/Pausable.sol";
 /// @notice defines some modifiers and utilities around interacting with Core
 abstract contract CoreRef is ICoreRef, Pausable {
     ICore private _core;
```

Comment:
While we're here, we should make core immutable as well to save on gas

Reply:
I'm tentatively down if we are sure we won't want to do core upgrades in the future

Reply:
I think core should be able to be set by governance